pkg/applicationserver/io/web/webhooks.go

```diff
@@ -336,25 +336,24 @@ func (w *webhooks) handleUp(ctx context.Context, msg *ttnpb.ApplicationUp) error
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			req, err := w.newRequest(ctx, msg, hook)
+			reqs, err := w.newRequests(ctx, msg, hook)
 			if err != nil {
 				logger.WithError(err).Warn("Failed to create request")
 				return
 			}
-			if req == nil {
-				return
-			}
-			logger.WithField("url", req.URL).Debug("Process message")
-			if err := w.target.Process(req); err != nil {
-				logger.WithError(err).Warn("Failed to process message")
+			for req := range reqs {
```

Comment:
Prefer `_, req := range reqs` here (same as PubSub).

Reply:
As we are only using index here, it gives the following error: 
```
should omit 2nd value from range; this loop is equivalent to `for req := range ...`
```

Reply:
Use `req` as value bellow, as opposed to `reqs[req]`.

Reply:
Corrected. Looks like this now:
```
for _, req := range reqs {
				logger.WithField("url", req.URL).Debug("Process message")
				if err := w.target.Process(req); err != nil {
					logger.WithError(err).Warn("Failed to process message")
				}
			}
```